Dshop/apps/products_catalogue/views.py

```diff
@@ -23,6 +23,19 @@ class ProductDetailView(DetailView):
     context_object_name = 'product'
     queryset = Product.objects.filter(is_active=True)
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        product = context['product']
+        attributes = product.get_attributes()
+        if product.parent_product:
+            product_variants = Product.objects.filter(parent_product=product.parent_product).exclude(
```

Comment:
Add is_active filter to all the products so it can be removed from template.

Reply:
done